### PR TITLE
Use stable releases

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -31,8 +31,6 @@
         "cakephp/utility": "Provides CakePHP security methods. Required for the JWT adapter and Legacy password hasher."
     },
     "license": "MIT",
-    "minimum-stability": "dev",
-    "prefer-stable": true,
```
Comment (dereuromark) [inline composer.json]: we can probably keep prefer stable right?

Comment (ADmad) [reply composer.json]: We can but it would be redundant without `"minimum-stability"` set to `dev` or `beta`

Comment (dereuromark) [reply composer.json]: Jep, but we have one less to always add and remove for each major.
Meh/^^

**Comment by dereuromark** [comment]
something is wrong with master. it is not the valid 2.x (cake4) code...
https://github.com/cakephp/authentication/compare/2.x?expand=1
thus the fails.

**Pull request closed without merging**